virttest/utils_test/libvirt.py

```diff
@@ -2317,6 +2317,7 @@ def create_tpm_dev(params):
     tpm_model = params.get("tpm_model", 'tpm-crb')
     backend_type = params.get("backend_type")
     backend_version = params.get("backend_version")
+    persistent_state = params.get("persistent_state", "no")
```

Comment:
persistent_state may not be set one default value if you want to use it as judgement point in line 2331.

Reply:
Done. Thanks. Actually it's not used by tpm_device.py, just update it for other cases using.